index.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Empty line at start of file?

Reply:
Added the lines and removed the ones at the top of the files